src/lib/dapp/index.ts

```diff
@@ -107,54 +113,31 @@ export class DAppConnector {
         projectId: this.projectId,
         metadata: this.dAppMetadata,
       })
-      const existingSessions = await this.checkPersistedState()
-
-      if (existingSessions.length) {
-        await this.onSessionConnected(existingSessions.pop()!)
+      const existingSessions = this.walletConnectClient.session.getAll()
```

Comment:
Could you please explain a bit why the logic for checking really alive sessions has been removed and simplified to this one?

Reply:
This function was designed to remove sessions when the corresponding wallet in the communication fails to respond. Non-responsiveness can occur if the wallet is closed or temporarily inaccessible; however, it is crucial that the session remains valid during such intervals. The getAll method from WalletConnect core is tasked with reinstating any session that should remain active, ensuring continuity and stability in the communication process.